clouddq/classes/dq_rule_binding.py

```diff
@@ -92,13 +92,14 @@ def from_dict(
         )
         if column_id:
             column_id.upper()
-        row_filter_id: str = get_from_dict_and_assert(
+        row_filter_ids: list[str] = get_from_dict_and_assert(
             config_id=rule_binding_id,
             kwargs=kwargs,
-            key="row_filter_id",
+            key="row_filter_ids",
```

Comment:
I suggest either 1) leaving this field as 'row_filter_id' for backwards compatibility or 2) allowing one-of 'row_filter_id' or 'row_filter_ids'. Perhaps (1) is preferable despite the unfortunate naming.

Reply:
i agree:) i fix it to allow both!